### ISSUE TITLE
Support of "shortcuts"

### ISSUE THREAD

**Issue by stritti** [ghys/habot#5]
I think there should be some mappings for short commands like:

* EN: "music louder" -> "increase music 10%"
* EN: "mute music" -> "set music to 0%"
* DE:  "Licht aus" -> "schalte das Licht aus"

**Comment by ghys** [comment]
Yeah OpenNLP is sometimes confused with short sentences like this.
I planned "increase" and "decrease" skills for later, as well as player controls ("previous track/next track/play/pause etc.").
*In theory* "Licht aus" should work after enough training ("`<OBJECT:object> Licht <END> aus`" and a couple other similar lines in deactivate-object.txt).
There is one implemented shortcut though: if you simply state the name of a tag without "object:" or "location:" it will simply interpret it as a get-status intent for the tag, that way you'll probably have controls to operate yourself. It work only work as long as you have visual feedback though (i.e. not audio only).

**Comment by stritti** [comment]
You are correct "Licht aus" should be a deactivate-intent.
On the other hand I found (especially in German), that you will talk/write in `telegram fashion`. I think some "shortcuts" or "aliases" could solve this because otherwise we have to add a lot attributes in the items-config.

One example (I translate 1:1 in worse English):
* "How warm is it in living room?" (Wie warm ist es im Wohnzimmer?) -> "give me the temperature in the living room"
